templates/user/auth/signin_inner.tmpl

```diff
@@ -48,7 +48,7 @@
 			</div>
 		</form>
 		{{end}}{{/*if .EnablePasswordSignInForm*/}}
-		{{$showOAuth2Methods := or .OAuth2Providers .EnableOpenIDSignIn}}
+		{{$showOAuth2Methods := or .OAuth2Providers .EnableOpenIDSignIn .EnableSSPI}}
```

Comment:
There is another `$showOAuth2Methods` in `signup_inner.tmpl`, would you like to update it together?